app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
```

Comment:
not an error, but a recommendation. It is more readable to write in this way:
```python
def __init__(
    self, 
    distance_from_city_center: float, 
    clean_power: int,
    average_rating: float, 
    count_of_ratings: int
) -> None:

Reply:
the same for every function and method where you have a lot of args and you can't take it in 1 line